tools/go.mod

```diff
@@ -121,46 +111,45 @@ require (
 	github.com/lufeee/execinquery v1.2.1 // indirect
 	github.com/magiconair/properties v1.8.6 // indirect
 	github.com/mailru/easyjson v0.7.6 // indirect
+	github.com/maratori/testableexamples v1.0.0 // indirect
```

Comment:
MIT - what is pulling this in?

Reply:
./github.com/golangci/golangci-lint/pkg/golinters/testpackage.go:       "github.com/maratori/testpackage/pkg/testpackage"
./github.com/golangci/golangci-lint/pkg/golinters/testableexamples.go:  "github.com/maratori/testableexamples/pkg/testableexamples"